nengo_gui/examples/tutorial/02-two-neurons.py

```diff
@@ -19,7 +19,7 @@
     stim = nengo.Node(0)
     ens = nengo.Ensemble(n_neurons=2, dimensions=1, seed=5)
     nengo.Connection(stim, ens)
-    
-    
+
+
     import time
-    slowdown = nengo.Node(lambda t: time.sleep(0.001)) 
\ No newline at end of file
+    slowdown = nengo.Node(lambda t: time.sleep(0.001))
```

Comment:
This line is also there in examples 00 and 01. My assumption is that the code is shown in case anyone needs to use this if graphics rendering is ahead of the simulation. I could add a comment in `00-intro.py` and remove these lines from 01 and 02?